### DISCUSSION TITLE
Changing the color and icon of a radiobutton (I think through pt?)

### DISCUSSION THREAD

**Discussion by RenzoDaal** [primefaces/community#1447] [PrimeVue] [answered]
This might be a stupid question but like it says in the title, I want to make 2 changes to my radiobutton:

1) The standard icon of the radiobutton is a circle which I want to change to a check sign.
2) The background color needs to be changed to green.

These changes only need to apply to the selected checkbox. The non-selected checkboxes need to remain their original style which is completely white with a grey border. I am guessing this has to be done with PT but unfortunately I couldn't figure it out.

**Comment by RenzoDaal** [comment]
My pt now looks like this, but the styling under input should only be applied to the selected radiobutton. I tried to use options.context.checked like it says in the documentation but that didn't work. Any suggestions?

```
:pt="{
  input: {
    style: {
      'border-color' : '#008000',
      'background-color' : '#ffffff'
    }
  },
  icon: {
    style: {
      'color' : '#008000',
      'font-size' : '0.9rem',
      'margin-top' : '4px',
      'font-weight' : 'bold'
    },
    class: 'pi pi-check'
  }
}
```

**Comment by tugcekucukoglu** [comment, marked as answer]
Using `box` instead of `input` could help.

```
:pt="{
        box: {
          style: {
            'border-color': '#008000',
            'background-color': '#ffffff',
          },
        },
        icon: {
          style: {
            color: '#008000',
            'font-size': '0.9rem',
            'font-weight': 'bold',
            'background-color': '#fff',
          },
          class: 'pi pi-check',
        },
      }"
```

**Reply by vladguan** [reply]
I am tryin to follow the above but when the radio is unselected, it leave a very small dot of the icon which I can't seem to get rid of. Is there a trick to it?
